### ISSUE TITLE
Include preprint reference in genetics portal evidence

### ISSUE THREAD

**Issue by d0choa** [opentargets/issues#1849]
We are currently including genetics portal evidence from GWAS studies in preprint.

An example of evidence for [CD226 and type I diabetes](https://main--platform-app.netlify.app/evidence/ENSG00000150637/EFO_0001359) is coming from [GCST90013791](https://www.ebi.ac.uk/gwas/studies/GCST90013791). This GWAS is part of a preprint publication at the moment linked in the GWAS catalog using its [DOI](https://doi.org/10.1101/2021.02.05.429962).

<img width="1090" alt="Screenshot 2021-11-12 at 15 59 08" src="https://user-images.githubusercontent.com/5097586/141496532-67e0645d-23b8-4bd8-88b0-7046552ce17b.png">

It would be good to keep track of the preprint reference in order to provide the appropriate credit to the authors of the study. This might eventually require some changes in the genetics portal study table.

**Comment by DSuveges** [comment]
Currently we have `literature` in which we are expecting a list of pmids, and `pmcIds`, which is a list of PMC identifiers. None of them works here.

We can either use the `urls` [field](https://github.com/opentargets/json_schema/blob/4b66e7fc8bbe42537238fc013aebd57edb85586c/opentargets.json#L1944) which is very flexible in terms of content. However if we can expect biorxiv identifiers it makes sense to create a new field. I would not go for a completely agnostic approach to support all DOIs, as I don't know how easy/difficult would it be to resolve any random identifiers. Also we have to make sure the GWAS data is propagated to Genetics Portal first.

**Comment by DSuveges** [comment]
After a discussion with Annalisa, it seems Genetics Portal won't ingest studies without pubmed id. It is quite strange how a single study got released in the 21.09 and 21.06 release:

```
+------------+----------+---------+
|     studyId|literature|projectId|
+------------+----------+---------+
|GCST90013791|      null|     GCST|
+------------+----------+---------+
```

In the current release there's no study (from the GWAS catalog) without pubmed ID, however there are many in the GWAS Catalog. eg [GCST90027775](https://www.ebi.ac.uk/gwas/studies/GCST90027775) however as the previous example shows, DOI pointing to biorxive might not available.

**Comment by d0choa** [comment]
Does this look like it could be closed, @DSuveges?
